### PR TITLE
Allow equal margin of timesheet card

### PR DESCRIPTION
**Pull request by jaeminlee0515**
Now it will allow for the margin's of the card to be equal.

Screenshot 2023-07-05 at 14-52-38 Gordon 360

It would now look like the above.

### CODE REVIEW

File: src/views/Timesheets/components/SavedShiftsList/index.jsx
Comment (amos-cha) [inline src/views/Timesheets/components/SavedShiftsList/index.jsx:1]: @EjPlatzer are we planning to still use 360 timesheets? if so, would it be worth it to refactor this to a functional component

**Review by amos-cha — COMMENTED** [review]
Looks good

File: src/views/Timesheets/components/SavedShiftsList/index.jsx
Comment (EjPlatzer) [reply src/views/Timesheets/components/SavedShiftsList/index.jsx:1]: > @EjPlatzer are we planning to still use 360 timesheets? if so, would it be worth it to refactor this to a functional component

No, I believe I told y'all this at the beginning of the summer when CTS learned it - we are finally moving forward with having student workers use Criterion for timesheets, so this code continues to be in maintenance-only mode.

Comment (amos-cha) [reply src/views/Timesheets/components/SavedShiftsList/index.jsx:1]: I remember this conversation ~sort-of.

**Pull request merged**